spec/requests/admin/user_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
```

Comment:
This spec is a little odd, I understand the rationale behind it, since the password line is optional, you want to test that workflow works. The issue though is that we don't actually use passwords in the website, so I think this test (and that line in the controller) are not needed